src/utilities/Dealer.cpp

```diff
@@ -1,21 +1,28 @@
-#include <roboteam_utils/Hungarian.h>
+/**
+ * The dealer will check for the flags that are set in plays, but also for the distance
+ * to a position that a robot might need to travel to. The lower the score of a robot, the better.
+ * The robot with the lowest score for a role will get assigned to that role, unless it has already been assigned.
+ * In that case it will pick the second best and so on.
```

Comment:
I don't think this is how the Hungarian works but I could be wrong. This sounds quite greedy, I thought the hungarian tries to find a more global solution

Reply:
Makes sense, my understanding of  the hungarian was wrong